pepper-apis/src/main/java/org/broadinstitute/ddp/db/UserDao.java

```diff
@@ -52,294 +42,18 @@
 
     private final String insertUserStmt;
     private final String hasAdminAccessQuery;
-    private final String profileExistsQuery;
-    private final String getUserIdFromGuid;
     private final String userClientRevocationQuery;
-    private final String patchGenderStmt;
-    private final String patchPreferredLanguageStmt;
 
     /**
      * Set up needed sql queries.
      */
     public UserDao(String insertUserStmt,
```

Comment:
Almost there! Soon we'll be able to remove `ddp.db.UserDao` and `ddp.db.UserDaoFactory`.

Reply:
May be can move remaining non profile related methods/code to other UserDao (ddp.db.dao.UserDao.java) and delete this one.

Reply:
Let's do that in a separate PR. I think those remaining methods are used in a few places, that will need cleanup.

Reply:
:+1: